### ISSUE TITLE
new patch vesrion to fix keycharm@0.2.0 peer dep warning

### ISSUE THREAD

**Issue by micahstubbs** [visjs/ngx-vis#426]
**Is your feature request related to a problem? Please describe.**

```py
npm i
npm WARN ngx-vis@3.1.0 requires a peer of keycharm@^0.2.0 but none is installed. 
You must install peer dependencies yourself.
```

**Describe the solution you'd like**

release v3.1.1 to fix keycharm@0.2.0 peer dep warning

**Additional context**

Other visjs ecosystem projects already depend on keycharm@^0.3.0, so the old 0.2.0 peer dep should not be necessary anymore.

**Comment by micahstubbs** [comment]
[blame](https://github.com/visjs/ngx-vis/blame/develop/package.json#L72)
[commit](https://github.com/visjs/ngx-vis/commit/bbb3fb7865c4437f4d5d3c9a0a05ef9d9d26a1fc)

**Comment by JavierT** [comment]
Hello,
when is planned the next release to solve this issue ? I still have the issue with the latest version and the only way to compile is to stay with node 16.14.0.
thanks!

**Comment by hypery2k** [comment]
this is fixed with v3.1.1
